### ISSUE TITLE
High contrast theme?

### ISSUE THREAD

**Issue by naught101** [calf-studio-gear/calf#226]
I love the calf plugins, they look great. However, sometimes I find the dark-grey-on-green theme a little straining on the eyes, and less clear than it could be. This is particularly the case for the Analyser, in spectrogram mode, where the low contrast makes things quite hard to read. 

I also prefer light-on-dark themes for most things, and the green backgrounds really stick out on my set-up. Would it be possible to add a high-contrast light-on-dark theme for the analyser and other spectrum graphs, and matching UI elements? Either the same green on dark, or possibly a blue on dark, to match the other UI elements?

**Comment by unfa** [comment]
I agree that the spectrogram's usability suffers in favor of the aesthetics.
